jms/src/main/scala/akka/stream/alpakka/jms/Jms.scala

```diff
@@ -84,6 +84,30 @@ final case class ConnectionRetrySettings(connectTimeout: FiniteDuration = 10.sec
   def withMaxBackoff(maxBackoff: Long, unit: TimeUnit): ConnectionRetrySettings =
     copy(maxBackoff = Duration(maxBackoff, unit))
   def withMaxRetries(maxRetries: Int): ConnectionRetrySettings = copy(maxRetries = maxRetries)
+  def withInfiniteRetries(): ConnectionRetrySettings = withMaxRetries(-1)
+
+  /** Hypothetical retry time, not accounting for maxBackoff. */
+  def waitTime(retryNumber: Int): FiniteDuration =
+    (initialRetry * Math.pow(retryNumber, backoffFactor)).asInstanceOf[FiniteDuration]
+}
+
+object SendRetrySettings {
+  def create(): SendRetrySettings = SendRetrySettings()
+}
+
+final case class SendRetrySettings(initialRetry: FiniteDuration = 50.millis,
+                                   backoffFactor: Double = 2,
+                                   maxBackoff: FiniteDuration = 10.seconds,
+                                   maxRetries: Int = 10) {
```

Comment:
I chose the default send retries to be much more tight than the reconnect retries. If other default settings make more sense, we should provide those.